ingest/ingest_pipeline.py

```diff
@@ -34,12 +34,20 @@
 
 # Ingest mtx files
 python ingest_pipeline.py --study-id 5d276a50421aa9117c982845 --study-file-id 5dd5ae25421aa910a723a337 ingest_expression --taxon-name 'Homo sapiens' --taxon-common-name human --matrix-file ../tests/data/mtx/matrix.mtx --matrix-file-type mtx --gene-file ../tests/data/genes.tsv --barcode-file ../tests/data/barcodes.tsv
+
+# Differential Expression analysis (dense matrix)
+python ingest_pipeline.py --study-id addedfeed000000000000000 --study-file-id dec0dedfeed1111111111111 differential_expression --annotation-name cell_type__ontology_label --annotation-type group --annotation-scope study --matrix-file-path ../tests/data/differential_expression/de_integration.tsv --matrix-file-type dense --annotation-file ../tests/data/differential_expression/de_integration_unordered_metadata.tsv --cluster-file ../tests/data/differential_expression/de_integration_cluster.tsv --cluster-name de_integration --study-accession SCPdev --differential-expression
```

Comment:
Nit: having a file named `de_integration.tsv` is a bit opaque.  Like `de_integration_unordered_metadata.tsv`, it'd be clearer to name the file something like `de_integration_dense_matrix`.  

(I could also see trimming out `integration` from all those file names, which seems like mostly filler.)

Reply:
`b709b84` agreed. I've updated the files names.